pkg/model/resources/nodeup.go

```diff
@@ -79,28 +81,28 @@ download-or-bust() {
   while true; do
     for url in "${urls[@]}"; do
       commands=(
-        "curl -f --compressed -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10"
-        "wget --compression=auto -O "${file}" --connect-timeout=20 --tries=6 --wait=10"
-        "curl -f -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10"
-        "wget -O "${file}" --connect-timeout=20 --tries=6 --wait=10"
+        "curl -f --compressed -Lo ${file} --connect-timeout 20 --retry 6 --retry-delay 10"
```

Comment:
I'm sort of surprised we don't need quotes here to keep shellcheck happy, but agree that we shouldn't need quotes in practice.

Reply:
Actually, with quotes is not working quite right...

Reply:
Right - I would guess we would need to get fancy with `'` and `"` and escaping